transbank-sdk-dotnet-rest-example/Controllers/TransaccionCompletaController.cs

```diff
@@ -1,28 +1,27 @@
 using System;
+using System.Web.Mvc;
 using System.Collections.Generic;
 using System.Linq;
-using System.Web.Mvc;
-using Transbank.Common;
 using Transbank.Webpay.TransaccionCompleta;
 
 namespace transbanksdkdotnetrestexample.Controllers
 {
     public class TransaccionCompletaController : Controller
     {
-
+        #region Transaccion Completa
```

Comment:
I think we are not using this annotations in the rest of the sdk.

Reply:
I saw it in other example like https://github.com/TransbankDevelopers/transbank-sdk-dotnet-webpay-rest-example/blob/335791e0b87eb7f98ca9e9ee9f94befc9b5aeb09/transbank-sdk-dotnet-rest-example/Controllers/WebpayPlusController.cs#L12

Reply:
Yeah, we should remove that too.